docs/cli/use-cases.md

```diff
@@ -58,129 +64,132 @@ The Wave CLI supports building a container from a list of [Conda][conda] package
 
 Conda builds support the following arguments:
 
-- `--conda-base-image`: A base image for installing Conda packages. The default value is `mambaorg/micromamba:1.5.10-noble`.
-- `--conda-channels`: One or more comma-separated channels. The default value is ` seqera,bioconda,conda-forge,defaults`.
-- `--conda-file`: A [Conda lock file][conda-lock]. Can be a local file or a URL to a remote file.
-- `--conda-package`: A Conda package to install. Can be specified multiple times. Expressions are supported, such as `bioconda::samtools=1.17` or `samtools>=1.0,<1.17`.
-- ` --conda-run-command`: A Docker `RUN` command used when containers are built. Can be specified multiple times.
+- `--conda-base-image`: Specifies the base image for installing Conda packages (default: `mambaorg/micromamba:1.5.10-noble`).
+- `--conda-channels`: Specifies one or more comma-separated channels (default: `seqera,bioconda,conda-forge,defaults`).
+- `--conda-file`: Specifies a [Conda lock file][conda-lock] path or URL.
+- `--conda-package`: Specifies a Conda package to install. Supports expressions such as `bioconda::samtools=1.17` or `samtools>=1.0,<1.17`. Can be specified multiple times.
```

Comment:
Does it accept comma-separated packages, or should it be specified in full multiple times if multiple packages are to be specified? Same question for `conda-run-command` below

Reply:
Not blocking, just worth clarifying in case we accept comma-separated.

Reply:
Good spot. Updating.